### ISSUE TITLE
Validation loss

### ISSUE THREAD

**Issue by Anurag-Swarnim-Yadav** [OpenNMT/OpenNMT-py#1900]
How can we get validation loss whenever we get validation accuracy?

Thanks 
Anurag

**Comment by francoishernandez** [comment]
You can have a look at the `report_manager` code:
https://github.com/OpenNMT/OpenNMT-py/blob/073428849c1d10dd4fae7f8fd92699cdc9f230a4/onmt/utils/report_manager.py#L158-L160
and the available attributes of the statistics object in `onmt.utils.statistics`.

**Comment by Anurag-Swarnim-Yadav** [comment]
So I have add self.log('Validation log : %g' % valid_stats.xent()) will do the job?

**Comment by francoishernandez** [comment]
Yes.
